src/core/http/http-client.ts

```diff
@@ -84,12 +83,18 @@ export class HttpClient {
     }
 
     public async post(url: string, body: any): Promise<any> {
+        const contentType = body instanceof FormData
+            ? "multipart/form-data"
+            : "application/json;charset=utf-8";
+        const requestBody = typeof body === "string" || body instanceof String || body instanceof FormData
+            ? body
+            : JSON.stringify(body)
```

Comment:
This methods was design to work only with `application/json` and the `postFile` method here to be used for `form-data`. Did we try only changing the usage to postFile instead of post before modifying this? Asking to know if there was any issue with the postFile implementation.

Reply:
Didn't test it. I compared the implementations between old and refactor code base, and this method looked pretty much what was required, and left it as is because it fixed the issue. We can also go back to it and use proper method